{"name": "困人处置列表-待处理", "status": "passed", "attachments": [{"name": "log", "source": "eae1fc57-fdae-48cf-a4b7-fea1ec5c1629-attachment.txt", "type": "text/plain"}], "start": 1692260235151, "stop": 1692260235377, "uuid": "6194c3bc-1d2c-4db9-87ae-07f418b38908", "historyId": "268ac27d44efa67b57ad425fd8b2df94", "testCaseId": "7335a6c89350efea2656ff5aefa50509", "fullName": "opsPC.test_emergency#test_device_error_trapped_list_002", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "16680-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}